### PR TITLE
Documentation about translation

### PR DESCRIPTION
**Pull request by nicosomb**
ping @wallabag/translators

### CODE REVIEW

File: docs/en/developer/translate.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Translate wallabag
+==================
+
+Translation file
```
Comment (tcitworld) [inline docs/en/developer/translate.rst]: *Translation files* (since there's two).

**Comment by tcitworld** [comment]
:+1:

**Comment by j0k3r** [comment]
I think we should write somewhere who translated what.

**Comment by nicosomb** [comment]
CONTRIBUTORS.MD ?

**Comment by j0k3r** [comment]
https://github.com/wallabag/wallabag/blob/v2/CREDITS.md

**Comment by tcitworld** [comment]
I really this this should be in the app itself, on the [http://v2.wallabag.org/about](about) page. We could make a translators and a libraries used tab.

**Comment by j0k3r** [comment]
@tcitworld :+1:

**Pull request merged**